### DISCUSSION TITLE
codespace not starting

### DISCUSSION THREAD

**Discussion by Veni380** [community/community#29505] [Codespaces] [answered]
This error message repeatedly pops up and I don't know what the issue is
![codespace](https://user-images.githubusercontent.com/99728207/184606512-78427c86-4832-437e-80cf-4ac63f2057a0.png)

**Comment by softwaredevelop** [comment, marked as answer]
try with [VS Code](https://code.visualstudio.com/) + [GitHub Codespaces extension](https://marketplace.visualstudio.com/items?itemName=GitHub.codespaces)

**Reply by Veni380** [reply]
yeah I setup my codespace on desktop and it did work, the problem was that code.cs50.io is down since I am doing that course
